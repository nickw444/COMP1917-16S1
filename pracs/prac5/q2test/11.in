11
-1 -6 10 -5 7 6 0 8 2 -4 4
