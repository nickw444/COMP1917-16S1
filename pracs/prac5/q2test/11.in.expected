average of array: 1.909091
